src/test/resources/application.conf

```diff
@@ -0,0 +1,8 @@
+postgres = {
```

Comment:
will this and the other application.conf eventually be removed from this repo and generated from firecloud-develop?

Reply:
No. As an independent service its config belongs here, not in `firecloud-develop`.

There's still an open question about how to configure independent services for deployment in FiaB. Options are maintaining copies in `firecloud-develop` or having FiaB config scripts pull config templates from independent service repos. Greg Polumbo has been working on getting Martha and Bond into FiaB and is, for now, maintaining separate copies in `firecloud-develop` just because it was the easiest to implement.